BootloaderCorePkg/Tools/BuildUtility.py

```diff
@@ -862,7 +862,11 @@ def check_for_git():
 
 def check_for_slimbootkeydir():
     if not os.path.exists(os.environ.get('SBL_KEY_DIR')):
-        raise Exception ("SBL_KEY_DIR is not valid. Set correct SBL_KEY_DIR path !!")
+        print ("!!! PRE_REQUISTE: PATH TO SBL_KEY_DIR HAS TO BE SET WITH SBL KEY DIRECTORY !!! \n"

```

Comment:
Fix the various typos: PRE-REQUISITE, AVAILABLE etc

Reply:
GenerateKeys script does not set the environment variable. 
Do we need to be pedantic here in our error message and provide example on how to set env var on Windows and Linux systesm?

Reply:
Fixed typos and updated messages to set env var